metadata-models/src/main/pegasus/com/linkedin/dashboard/DashboardUserUsageCounts.pdl

```diff
@@ -29,4 +29,10 @@ record DashboardUserUsageCounts {
   @TimeseriesField = {}
   usageCount: optional int
 
+  /**
+   * If user_email is set, we attempt to resolve the user's urn upon ingest
+   */
+  @TimeseriesField = {}
+  userEmail: optional string
```

Comment:
nitpick: we can just name this "email" since the context of "user" is already provided in the model (user usage counts) 

Not a blocker. I'm happy to ship and merge once CI is green

Reply:
Also, just saw that this is good for consistency with DatasetUserUsageCount. Let's leave as is